src/tests/class_tests/openms/source/MQEvidenceExporter_test.cpp

```diff
@@ -65,33 +68,47 @@ END_SECTION
 /////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////
 START_SECTION((void exportFeatureMap(
-                    const OpenMS::FeatureMap &feature_map,
-                    const OpenMS::ConsensusMap &cmap)))
+                    const OpenMS::FeatureMap& feature_map  TEST_FILE_SIMILAR(filename.c_str(), OPENMS_GET_TEST_DATA_PATH("evidence_.txt"));,
```

Comment:
hier stimmt was nicht. In der Funktionssignatur sollte `TEST_FILE_SIMILAR` nicht vorkommen :)